src/components/progress.scss

```diff
@@ -0,0 +1,133 @@
+// * Progress Bar component
```

Comment:
I wonder if this should be capitalized `Progress` or `ProgressBar` to be consistent with the other components. There are a few that aren't capped but these are legacy from prior to when we changed the convention for to prevent conflicts when importing into LFE.

Reply:
Not sure exactly what you mean here. Just change it in the comment? I tried to follow the pattern of the others, but let me know what I should change it to.

Reply:
Sorry.. I just meant the file name.

Reply:
Got it!